.gitignore

```diff
@@ -31,6 +31,7 @@ site/
 token.json
 .env
 credentials.json
+jira_tokens.db
```

Comment:
You still have to remove the file from the repository if you have already committed it.